src/GeoJson/Feature/Feature.php

```diff
@@ -13,60 +18,51 @@
  */
 class Feature extends GeoJson
 {
-    protected $type = 'Feature';
+    protected string $type = 'Feature';
 
-    /**
-     * @var Geometry
-     */
-    protected $geometry;
+    protected ?Geometry $geometry;
 
     /**
      * Properties are a JSON object, which corresponds to an associative array, or null.
      *
      * @see https://www.rfc-editor.org/rfc/rfc7946#section-3.2
-     *
-     * @var array|null
      */
-    protected $properties;
+    protected ?array $properties;
 
     /**
-     * @var mixed
+     * The identifier is either a JSON string or a number.
+     *
+     * @see https://www.rfc-editor.org/rfc/rfc7946#section-3.2
```

Comment:
Noted that this relates to the comment in the OP about potentially enforcing these types. Thanks for cross-linking the RFC here.